phpBB/styles/prosilver/template/ucp_reset_password.html

```diff
@@ -10,7 +10,7 @@ <h2>{{ lang('RESET_PASSWORD') }}</h2>
 
 		<fieldset>
 		{% if S_IS_PASSWORD_RESET %}
-			{% if PASSWORD_RESET_ERRORS %}<p class="error">{{ PASSWORD_RESET_ERRORS | join('<br>') }}</p>{% endif %}
+			{% if PASSWORD_RESET_ERRORS %}<p class="error">{{ PASSWORD_RESET_ERRORS | join('<br>') | raw }}</p>{% endif %}
```

Comment:
Can you explain why you need `raw`?

Reply:
Cause it looks fine without it:
![image](https://user-images.githubusercontent.com/394418/72201011-14d8c500-3450-11ea-8ed3-64e0b71033fe.png)

Reply:
I have encountered a situation where without `raw` rendered as `<br>`, unable to replicate it at the moment though, moreover... the following 3 examples can be used, I opted for that one.

```
join('<br>')
join('<br>') | raw
join('\n') | nl2br
```

Reply:
Please remove raw then.